pmda/rmsf.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import MDAnalysis as mda
+import pmda
+import dask
+from dask.delayed import delayed
+import time
+from joblib import cpu_count
```

Comment:
Remove this line.

Reply:
Do not have un-necessary imports: check that each module or object is being used.

You can also run the PEP8 tests and they should complain.
```
pytest --pep8 
```
(or something similar, from the top level of the checked out pmda code repo)

Reply:
See also the output of the linter in the lint test https://travis-ci.org/MDAnalysis/pmda/jobs/550007184. You can run it locally with
```
pylint pmda
```
Fixe everything that PEP8 (Python style) and pylint (general coding style and constructs) complain about.